tools/diagnostics/base_tech_bundle/__init__.py

```diff
@@ -1,18 +1,98 @@
 from service_bundle import ServiceBundle
 
+import json
+import logging
+import config
+import sdk_cmd
+
+log = logging.getLogger(__name__)
+
 
 class BaseTechBundle(ServiceBundle):
+
+    def __init__(self, package_name, service_name, scheduler_tasks, service, output_directory):
+        self.package_name = package_name
+        self.service_name = service_name
+        self.scheduler_tasks = scheduler_tasks
+        self.service = service
+        self.framework_id = service.get("id")
+        self.output_directory = output_directory
+        self.install_cli()
+
+    @config.retry
+    def install_cli(self):
+        sdk_cmd.run_cli(
+            "package install {} --cli --yes".format(self.package_name),
+            print_output=False,
+            check=True,
+        )
+
+    @config.retry
+    def create_configuration_file(self):
+        rc, stdout, stderr = sdk_cmd.svc_cli(
+            self.package_name, self.service_name, "describe", print_output=False
```

Comment:
(nit) This would be a subset of `dcos marathon app show ....`, is the app definition already captured somewhere (or may be do you think it would be useful to capture the app definition) ?